### ISSUE TITLE
Feature Request: Variable Frameskip

### ISSUE THREAD

**Issue by KMFDManic** [libretro/yabause#250]
Currently, the implementation of Frameskip, as Auto or None doesn't work as well as it could potentially work, with Dynarec being ignored, obviously.  As Auto, it is more of a fast slide show, rather than smooth like when it is Off.  If Variable Numbers are added, say, at least, 0-6, and Auto, it would make a big difference as far as platforms that cannot run Yaba with Dynarec.  If any of you have any suggestions on the best way to handle this, I can test it on my fork, then pull request it once I have solid test results.  0-6 should be sufficient to cover most bases before going Auto.  Thanks:)

**Comment by barbudreadmon** [comment]
Not sure which core you are talking about, but afaik all yabause-based cores have a working frameskip option.

**Comment by KMFDManic** [comment]
@barbudreadmon I ended up being able to pull off what I wanted to do.  On "higher" platforms, with normal Yabause, NOT Yabasanshiro, games suffer really bad because of not having dynamic recompiler.  I merely wanted to try to better the way the execution of frameskipping currently works.  I see that the renderer doesn't play all too well with it, based on previous commits.  Thanks for reading and responding.  I got what I needed done, for the time being!  And, by the way, exceptional work on all of the commits you consistently put out!
